docs/pages/preview/upcoming-releases.mdx

```diff
@@ -40,24 +40,24 @@ You can find the full list of fixes and features in the [Github milestone](https
 | ------- | ----------------- | ------------------- |
 | Release | August 30th, 2021 | Production release. |
 
-## Teleport Cloud "Washington"
+## Teleport Cloud
 
-The Cloud team improves user experience - adds databases and kubernetes user interfaces
-and wizards, adds downloads page.
-The team improves cloud self service with password recovery and cancellation, adds data export.
+The early 2022 focus for the Teleport Cloud product is to focus on 3 key areas:
+- Ensuring the best routing and latency experience
+- Stronger outbound communications on cloud health and infrastructure changes
+- UX where the cloud offering differs from the on-prem offering
 
 ### Release Schedule
 
-The cloud team ships the release features as they are ready up to July 15th.
+| Date | Description |
+| - | - |
+| March 2022 | Complete rollout of our [worldwide access design](https://goteleport.com/blog/access-proxy-latency/), allowing low-latency access from around the world. |
+| April 2022 | Update security audit from independent auditors. [2021 Audit](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/) |
+| Q1 2022 | Rollout technical communications framework, notifying technical contacts of Planned Maintenance, Infrastructure Changes and Outages |
```

Comment:
- "Rollout" should be "Roll out a"
- I think we can probably lowercase "Planned Maintenance, Infrastructure Changes and Outages" unless these are official terms for things that need to be uppercased.